dashboard/app/controllers/registrations_controller.rb

```diff
@@ -236,9 +240,10 @@ def set_email
   def set_parent_email
     return head(:bad_request) if params[:user].nil?
 
-    successfully_updated = update_parent_email
+    successfully_updated = current_user.update_without_password(parent_email_params)
 
     if successfully_updated
+      ParentMailer.parent_email_added_to_student_account(current_user.parent_email, current_user).deliver_now
```

Comment:
It looks like this is being in two place now for similar reasons? Would it work to have an `after_save` on `user.rb` which sends the email `if parent_email_changed?` ?

Reply:
I don't think that will work because there's another way to change `parent_email` that has a different email sent. I'd love to combine them at some point but I think for now we have to differentiate these cases somehow.